autobahn/twisted/wamp.py

```diff
@@ -84,7 +85,29 @@ def _reject_future(future, error):
 
     @staticmethod
     def _add_future_callbacks(future, callback, errback):
-        return future.addCallbacks(callback, errback)
+        # callback and/or errback may be None
+
+        # Note that errback takes 3 args: type, exception, tb our
+        # errbacks can be consistent between Twisted and asyncio.
```

Comment:
I'd probably lean towards the "Failure-ish thing for asyncio case", but I'm probably really biased since I have more Twisted than asyncio experience ;)